scripts/configurator/parser/test_cmakeparser.py

```diff
@@ -1,5 +1,6 @@
-from cmake_parser import cmake_parser
+from re import L
 from unittest import TestCase
+from cmake_parser import lines_to_rows, configurator_type, write_file, row_type
```

Comment:
PEP 8 Heavily discourages that, because it's not esplicit and it can lead to circular imports

Reply:
Well, I believe that applying blindly even very good coding standards is not the best. `from X import *` can cause problems indeed, but makes sense when you're literally importing _every_ single function and class from a module, especially for simple and small modules like that one. It's not like doing `using namespace std;` in C++. It's like using your local mini-namespace defined in a another file in the same directory. Indeed, for small C++ projects we use the same namespace for all the files. Just, in this case Python automatically creates a new namespace for each file. And... about the cycling dependencies: I actually appreciate the fact that `import *` will fail in this case, because it will force me to avoid the cyclic dependency. When possible, it's _much better_ to avoid having 2 files each one of them depending on the other. When that happens, it is a _strong sign_ that maybe those file should be just merged into one, as the coupling is too tight for them to be separate.

Also, the scale matters: what is bad practice for the medium-large scale, can be perfectly acceptable or even _good_ for the small-scale projects. The [FizzBuzzEnterpriseEdition](https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition) project is a good example of what happens when some mostly good coding practices used in enterprise large-scale are forced in small-scale project. 

Still, I won't insist further on this point, as there is no damage in the extra verbosity. Feel free to keep the explicit list of imports. I just hope that I made you at least think more about the problem and all the implications. Coding standard rules are generally not written in stone, but are _guidelines_ generally designed for medium-large projects.